examples/mini-docusaurus/server.js

```diff
@@ -23,9 +27,27 @@ app.get('*', function(req, res) {
       </head>
       <body>
         ${convertedFile}
+        <script src="http://localhost:${PORT}/livereload.js"></script>
       </body>
     </html>
   `);
 });
 
 app.listen(5000);
+
+const server = tinylr();
+server.listen(PORT, function() {
+  console.log('Livereload listening on %s ...', PORT);
+})
+
+// gaze watches some specified files and triggers a callback when they change.
+gaze('docs/**/*', function () {
```

Comment:
Part 3 - Start file watching process and when there's a change, tell the LiveReload server which will then tell the client to refresh itself.